### PR TITLE
update README and Makefile

### PR DESCRIPTION
**Pull request by sallyom**
updates Makefile to allow a custom image registry like so:
`make IMAGE_REGISTRY=quay.io/sallyom images`

updates README to link common operator/operand info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,26 @@
 # cluster-authentication-operator
+The authentication operator is an 
+[OpenShift ClusterOperator](https://github.com/openshift/enhancements/blob/master/enhancements/operator-dev-doc.md#what-is-an-openshift-clusteroperator).    
+It installs and maintains the Authentication [Custom Resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) in a cluster and can be viewed with:
```
Comment (stlaz) [inline README.md]: It is only partially maintaining the CR, but the main purpose of the CR is to be one of the configuration entrypoints for the operator.

`s/and can be viewed with/which can be viewed with` -> the original suggests the operator can be viewed by the command

File: README.md [hunk 2]
```diff
@@ -1,6 +1,26 @@
 # cluster-authentication-operator
+The authentication operator is an 
+[OpenShift ClusterOperator](https://github.com/openshift/enhancements/blob/master/enhancements/operator-dev-doc.md#what-is-an-openshift-clusteroperator).    
+It installs and maintains the Authentication [Custom Resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) in a cluster and can be viewed with:     
+```
+oc get clusteroperator authentication -o yaml
+```
+
+The [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
+`authentications.operator.openshift.io`
```
Comment (stlaz) [inline README.md]: `definition of`

File: README.md [hunk 3]
```diff
@@ -1,6 +1,26 @@
...
+```
+oc get clusteroperator authentication -o yaml
+```
+
+The [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
+`authentications.operator.openshift.io`    
+can be viewed in a cluster with:
+
+```console
+$ oc explain authentications.operator.openshift.io
```
Comment (stlaz) [inline README.md]: That's a human-readable description of the CR's fields, but the CRD can be viewed by `oc get crd authentications.operator.openshift.io -o yaml`

Comment (sallyom) [reply README.md]: ok i'll update that, and same, i am always confused by `authentication.{config,operator}`

Comment (sallyom) [reply README.md]: updated

**Pull request merged**